cartographer/mapping/trajectory_builder_interface.h

```diff
@@ -69,4 +75,4 @@ class GlobalTrajectoryBuilderInterface {
 }  // namespace mapping
```

Comment:
Drive-by: style: type declaration (the callback) comes before all other declarations.

Reply:
Done.